master/internal/api_config_policies.go

```diff
@@ -149,13 +150,13 @@ func (a *apiServer) PutWorkspaceConfigPolicies(
 		Constraints:     constraints,
 	})
 	if err != nil {
-		return nil, fmt.Errorf("error setting task config policies: %w", err)
+		return nil, status.Errorf(codes.Internal, settingPoliciesErr+err.Error())
```

Comment:
do we need a colon after `settingPoliciesErr`? Or is this taken care of with the string concat?

Reply:
done